roles/openshift_preflight/facts/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: set common variables
+  set_fact:
+    deployment_type: "{{ deployment_type | default('openshift-enterprise') }}"
```

Comment:
In general, we usually want to default to origin if not specified. It might be better to expect the user to set this (and if we use a container or some other method of distribution we override the value there).